src/goblintCil.ml

```diff
@@ -0,0 +1,45 @@
+include Cil
+(** @inline *)
+
+(** {1 CIL modules} *)
+
+module Cfg = Cfg
+module Check = Check
+module Cil = Cil
+module Cilint = Cilint
+module Cillower = Cillower (* ? *)
```

Comment:
As there are comments about using this module in the documentation of `CEnum`, I would expose this.